gdsfactory/path.py

```diff
@@ -1139,7 +1147,11 @@ def euler(
     Rp = R0 / (np.sqrt(p * alpha))
     sp = R0 * np.sqrt(p * alpha)
     s0 = 2 * sp + Rp * alpha * (1 - p)
-    npoints = abs(int(npoints * angle / 360))
+
+    PDK = get_active_pdk()
+    npoints = npoints or abs(int(angle / 360 * radius / PDK.bend_points_distance / 2))
```

Comment:
is this a good enough approximation for an euler bend? I do this (original xy is the curve with a conservatively high point count)
```python
xy = np.asarray(sg.LineString(xy).simplify(
        tolerance=precision * 0.5 * radius / (radius + cross_section.width + cladding_buffer),
        preserve_topology=False).coords)
```

Reply:
ah, just noticed the lines below it. nice

Reply:
we can improve this later, and maybe move it to gdstk.Flexpath